storage/aws/aws.go

```diff
@@ -213,7 +221,7 @@ func (s *Storage) updateCP(ctx context.Context, newSize uint64, newRoot []byte)
 	if err != nil {
 		return fmt.Errorf("newCP: %v", err)
 	}
-	if err := s.objStore.setObject(ctx, layout.CheckpointPath, cpRaw, nil, ckptContType); err != nil {
+	if err := s.objStore.setObject(ctx, layout.CheckpointPath, cpRaw, false, ckptContType); err != nil {
```

Comment:
For readability, do you think it's worth adding named consts for this new bool param?